honeybadgermpc/progs/fixedpoint.py

```diff
@@ -23,8 +23,11 @@
 
 F = 32  # The precision (binary bits)
 """
-This implementation of the library is not completely hiding. This leaks information about the bits used in computation which is determinied by the security parameter Kappa.
-In particular, we leak O(1/(2^Kappa)) information theorotic bits per operation on a floating point secret.
+This implementation of the library is not completely hiding. This leaks
+information about the bits used in computation which is determinied by
+the security parameter Kappa.
+In particular, we leak O(1/(2^Kappa)) information theorotic bits per
+operation on a floating point secret.
```

Comment:
This comment looks wrong. What does it mean to leak an information theoretic bit? This is fixed point not floating point. This needs attention